src/components/panel/index.jsx

```diff
@@ -283,33 +309,27 @@ export default class Panel extends Component {
             userEmail={userEmail} />
         }
 
-        { !!isUserAuthenticated &&
-          <SubmitComment
-            userComment={userComment}
-            onUserCommentChange={this.onUserCommentChange}
-            onCommentSubmit={this.onCommentSubmit}
-          />
-        }
-
-        { !!comments &&
+        { !!isUserAuthenticated && !!comments &&
           <Comments
             onUserCommentDelete={this.onUserCommentDelete}
             currentUser={userEmail}
             comments={comments}
           />
         }
 
+        { !!isUserAuthenticated &&
```

Comment:
Can we move the 'add comment' section to the top for now? The comments are in descending order (last comment shows first atm) so if we swap this around we'll need to change the API to re-order the comments in ascending order.